src/cr/cube/cubepart.py

```diff
@@ -239,6 +239,8 @@ class _Slice(CubePartition):
     dimensions which can be crosstabbed in a slice.
     """
 
+    z_alpha = 1.959964
```

Comment:
Is this the confidence level, or better is this the zscore value given the confidence level = 95%? If yes the name `z_alpha` seems fishy/shady to me. Can we try to find a better name?